src/SFA.DAS.Commitments.Application/Services/CohortApprovalService.cs

```diff
@@ -26,22 +28,25 @@ internal class CohortApprovalService
         private readonly IMediator _mediator;
         private readonly OverlappingApprenticeshipService _overlappingApprenticeshipService;
         private readonly ApprenticeshipEventsService _apprenticeshipEventsService;
+        private readonly IApprenticeshipInfoService _apprenticeshipInfoService;
         private readonly ICommitmentsLogger _logger;
 
-        internal CohortApprovalService(IApprenticeshipRepository apprenticeshipRepository,
+        public CohortApprovalService(IApprenticeshipRepository apprenticeshipRepository,
```

Comment:
I can't see why this has changed to `public`, there are no new public funcs. Maybe before trying `InternalsVisibleTo()`?